src/controllers/OtherReasonController.ts

```diff
@@ -33,12 +34,15 @@ class FormSubmissionProcessor extends UpdateSessionFormSubmissionProcessor<Other
     }
 
     protected prepareModelPriorSessionSave(appeal: Appeal, value: OtherReason): Appeal {
```

Comment:
I am not sure whether we need to log result of `prepareModelPriorSessionSave` since you are logging calls to `UpdateSessionFormSubmissionProcessor` which logs session object prior storing it in Redis. Data will l be same resulting in duplicates. What matters is really what was saved into session.

Reply:
It's probably best to remove the log in UpdateSessionFormSubmissionProcessor because we want to know which controller triggered the save